### PR TITLE
Removed remoteQuery (MOB-814) - replaced with GNSClientInternal

### PR DESCRIPTION
**Pull request by westyUMass**
Fixed Eclipse compiler warnings for private methods (MOB-1082)
Also fixed an ACL issue with VerifyAccount.

### CODE REVIEW

**Comment by westyUMass** [comment]
This pull request is still outstanding.

**Comment by akarthik10** [comment]
@westyUMass , are you blocked because of this PR? The trivial warning changes look fine to me but I think I need to understand few things in greater detail more before approving this. This may take some time.

**Comment by westyUMass** [comment]
Not blocked, for the next few days at least. I just wanted to get this merged before any other merges were done so no conflicts were generated. The big change is the introduction of deleteOrNotExists and it's use to replace uses of the RemoteQuery class. All of the stuff in ServerIntegrationTest is just syntax, formatting and warning removal.

File: build.xml [hunk 1]
```diff
@@ -74,7 +74,7 @@
     <javac compiler="org.eclipse.jdt.core.JDTCompilerAdapter" source="1.8"
          target="1.8" srcdir="${src.dir}:${test.dir}" destdir="${build.classes.dir}" debug="true" debuglevel="lines,vars,source" includeantruntime="false">
       <classpath refid="classpath.base"/>
-      <compilerarg value="-warn:+allJavadoc" />
+      <compilerarg value="-warn:+allJavadoc,+invalidJavadocVisibility(public),+missingJavadocTagsVisibility(public)" />
```
Comment (westyUMass) [inline build.xml]: maybe it should be -invalidJavadocVisibility(public) 

..."specifying the given option with "+" will enable more warnings than the default, and specifying "-" disables some that are enabled by default"

Comment (akarthik10) [inline build.xml]: I've been trying variants of that from 30 minutes with no luck.

**Comment by akarthik10** [comment]
Before I merge this, what would you like to do to `invalidJavadocVisibility`? It is harmless but I haven't yet found a way to make it work.

**Comment by westyUMass** [comment]
I pulled out the useless flags. Until +allJavadoc is fixed or removed we'll have a bunch of spurious warnings.

**Pull request merged**